defaults/main.yml

```diff
@@ -11,6 +11,8 @@ disable_udf: yes
 disable_fat: yes
 temp_dir_size: 1G
 disable_squashfs: no #Disabling squashfs will prevent the use of snap
+# 1.1.24 Disable USB Storage | disables the kernel modules that are using usb storage
+disable_usb_kernel_modules: true
```

Comment:
Better to set it to false by default, I think. Users can then override it as they wish.

Reply:
BTW, just saw that the `disable_usb` parameter is set only in the parameters of section 6 (#189), but it's also already used in section 1. Could you move this parameter upwards?

Reply:
done